### DISCUSSION TITLE
Support of Green Power devices

### DISCUSSION THREAD

**Discussion by nickagian** [Koenkk/zigbee2mqtt#11027] [General]
I've read through the relevant issue thread here and I'm still unsure how to use Green Power devices with Z2M. It is written that a prerequisite is that a Hue device has to be in range, but no further details are given.

Will any Hue device work? Can I have for example a lamp?
Do I need a Hue gateway?
Do I need to do anything special or just power the Hue device and keep it in range?

**Comment by Koenkk** [comment]
> Will any Hue device work? Can I have for example a lamp?

Any mains powered device will work

> Do I need a Hue gateway?

no

> Do I need to do anything special or just power the Hue device and keep it in range?

Just keep it in range
